packages/react-component-library/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -26,12 +23,8 @@ stories.add('Danger', () => {
     <Dialog
       title="Dialog Title"
       description="Dialog description."
-      onConfirm={event => {
-        console.log('onConfirm')
```

Comment:
I started off using action then switched to log because actions didn't show up but that is likely because it wasn't configured right and I was lazy :) 👍